cypress/fixtures/postcodes.json

```diff
@@ -1,4 +1,5 @@
 {
   "default": "B12JP",
+  "test": "L1 0BG",
```

Comment:
Wonder if we can use something more explicit here as a key?

Reply:
Changed this to defaultPostcode

Reply:
This is not needed as if you go to/courses/maths on QA and use B12JP you will get courses.